### PR TITLE
Update to scipp 0.16.3 and scippneutron 0.9.3

### PR DESCRIPTION
**Pull request by jl-wynen**
Failing because of a bug identified in scipp and fixed in https://github.com/scipp/scipp/pull/2821

### CODE REVIEW

File: .buildconfig/ci.yml [hunk 1]
```diff
@@ -223,24 +240,25 @@ dependencies:
...
+  - qt=5.12.9
   - rapidjson=1.1.0
   - readline=8.1.2
   - requests=2.28.1
-  - scipp=0.16.2
-  - scippneutron=0.9.2
-  - scippnexus=0.1.8
-  - scipy=1.9.0
-  - setuptools=65.3.0
+  - scipp=0.16.3
```
Comment (SimonHeybrock) [inline .buildconfig/ci.yml]: Won't you need 0.16.4?

**Review by SimonHeybrock — COMMENTED** [review]
Note: `pip-compile-multi` has an option to only update a specific package. We should generally use that when targeting a specific fix, I think?

File: .buildconfig/ci.yml (hunk 1 above)
Comment (jl-wynen) [reply .buildconfig/ci.yml]: Yes, once that actually exists.

**Comment by jl-wynen** [comment]
> Note: `pip-compile-multi` has an option to only update a specific package. We should generally use that when targeting a specific fix, I think?

This uses conda, not pip-compile. Is there such an option, too?

**Comment by SimonHeybrock** [comment]
> > Note: `pip-compile-multi` has an option to only update a specific package. We should generally use that when targeting a specific fix, I think?
> 
> This uses conda, not pip-compile. Is there such an option, too?

Hmm, probably not.

**Review by SimonHeybrock — APPROVED** [review]

**Review by nvaytet — APPROVED** [review]

**Pull request merged**